lib/components/html.js

```diff
@@ -1 +1,2 @@
-module.exports = require('./dist/esm/html');
+// eslint-disable-next-line import/no-unresolved
+export * from './dist/esm/html';
```

Comment:
@ndelangen these three "entrypoints" (`@storybook-components/html`, `@storybook-router/utils` and `@storybook-theming/create`) don't actually work now we pre-bundle. I included the change just to call it out in the PR, but right now these files don't point to anything (thus the linting issue)

Reply:
hmm, that seems like a pretty big deal.. I do not remember why we had those separate  entrypoints to begin with, do you remember why we introduced those?\

Reply:
Nope, I know nothing about them. @shilman?

Reply:
@tmeasday @ndelangen 

`@components/html` was a grouping for MDX docs elements. we deprecated it a year ago, so if we have an unintentional breaking change there it's not the end of the world:

https://github.com/storybookjs/storybook/blob/next/lib/components/src/html.tsx

I'm less sure about the other two. @ndelangen I think that's more your territory.